tendrl/node_agent/objects/cpu/__init__.py

```diff
@@ -0,0 +1,28 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.node_agent.manager.pull_hardware_inventory import getNodeCpu
```

Comment:
put these "Get$Object" (eg: getNodeCpu) methods in these respective objects, pull_hardware_inventory needs to be removed.